projectDocs/dev/developerGuide/developerGuide.t2t

```diff
@@ -781,20 +781,31 @@ This manifest file contains key = value pairs declaring info such as the add-on'
 Although it is highly suggested that manifests contain all fields, the fields marked as mandatory must be included.
 Otherwise, the add-on will not install.
 
-- name (string): A short unique name for the add-on. This is used to differentiate add-ons internally and is not shown to the user. (Mandatory)
-- summary (string): The name of the add-on as shown to the user. (Mandatory)
-- version (string): The version of this add-on; e.g. 2.0. (Mandatory)
-- author (string): The author of this add-on, preferably in the form Full Name <email address>; e.g. Michael Curran <mick@kulgan.net>. (Mandatory)
+- name (string, required): A short unique name for the add-on.
+This is used to differentiate add-ons internally and is not shown to the user.
+The recommended convention is lowerCamelCase.
+Special characters should be avoided.
+- summary (string, required): The name of the add-on as shown to the user.
+- version (string, required): The version of this add-on; e.g. 2.0.1.
+When uploading to the Add-on Store certain requirements apply:
+  - Using ``<major>.<minor>.<patch>`` format.
+  - For a user to be able to update to this add-on, the version must be greater than the last version uploaded.
+  - Add-on versions are expected to be unique for the addon name and channel, meaning that a beta, stable and dev version of the same add-on cannot share a version number.
+  This is so there can be a unique ordering of newest to oldest.
+  - The suggested convention is to increment the patch version number for dev versions, increment the minor version number for beta versions, and increment the major version number for stable versions.
+  -
+- author (string, required): The author of this add-on, preferably in the form Full Name <email address>; e.g. Michael Curran <mick@example.com>.
 - description (string): A sentence or two describing what the add-on does.
 - url (string): A URL where this add-on, further info and upgrades can be found.
+This should start with ``https://``.
 - docFileName (string): The name of the main documentation file for this add-on; e.g. readme.html. See the [Add-on Documentation #AddonDoc] section for more details.
-- minimumNVDAVersion (string): The minimum required version of NVDA for this add-on to be installed or enabled.
+- minimumNVDAVersion (string, required): The minimum required version of NVDA for this add-on to be installed or enabled.
```

Comment:
I thought it had to be one of the listed API versions, not just a minimum NVDA version.
E.g. as i recall, many of us wanted to use 2017.3 as a minimum version, but this was disallowed. According to the proposed text it should again be possible.

Reply:
These are only recommendations, so you can still use `2017.3` it just cannot be submitted to the store sadly.